app/assets/javascripts/components/uploads/uploads_handler.jsx

```diff
@@ -13,18 +13,19 @@ const UPLOADS_PER_PAGE = 100;
 
 const UploadsHandler = createReactClass({
   displayName: 'UploadsHandler',
-
   propTypes: {
     course_id: PropTypes.string,
     course: PropTypes.object
   },
 
   getInitialState() {
+    const length_selecteduploads = this.props.selectedUploads.length;
```

Comment:
For style consistency, this should be done with camelCase, like `selectedUploadsLength`.

Alternatively you could just do the whole pageCount calculation inside the return object, without the intermediate const.

Reply:
@ragesoss  Done !